samples/SubscriptionsGraphQLServer/SubscriptionExample/SubscriptionExample/Startup.cs

```diff
@@ -0,0 +1,73 @@
+﻿using GraphQL;
+using GraphQL.Conventions;
+using GraphQL.Conventions.Adapters;
+using GraphQL.Conventions.Builders;
+using GraphQL.Http;
+using GraphQL.Server;
+using GraphQL.Server.Internal;
+using GraphQL.Server.Transports.Subscriptions.Abstractions;
+using GraphQL.Server.Transports.WebSockets;
+using GraphQL.Server.Ui.Playground;
+using GraphQL.Types;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using SubscriptionExample.Core;
+
+namespace SubscriptionExample
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var typeAdapter = new GraphTypeAdapter();
+            var constructor = new SchemaConstructor<ISchema, IGraphType>(typeAdapter);
+            var schema = constructor.Build(typeof(SchemaDefinition<GraphQl.Query, GraphQl.Mutation, GraphQl.Subscription>));

```

Comment:
I've got some refactoring in progress which will tidy up this initialisation code. Just leave as-is for now, and I will address in a follow-up PR